examples/VLCB_1in1out/VLCB_1in1out.ino

```diff
@@ -82,10 +82,11 @@ VLCB::Configuration modconfig;               // configuration object
 VLCB::CAN2515 can2515;                  // CAN transport object
 VLCB::MinimumNodeService mnService;
 VLCB::CanService canService;
-VLCB::CbusService cbusService;               // service for CBUS op-codes
 VLCB::EventConsumerService ecService;
 VLCB::EventTeachingService etService;
-VLCB::Controller controller(&userInterface, &modconfig, &can2515, { &mnService, &canService, &cbusService, &ecService, &etService }); // Controller object
+VLCB::CbusService cbusService;               // service for CBUS op-codes
+VLCB::Controller controller(&userInterface, &modconfig, &can2515, 
+                            { &mnService, &canService, &ecService, &etService, &cbusService }); // Controller object
```

Comment:
Wrap long line. Also put cbusService last as it is a catch-all service.

Reply:
I was wondering about this earlier.  This line keeps growing as we add more services.  Can we put each service within the braces on its own line? This would make it easier for a user to delete or comment out those they don't need.

Reply:
I was thinking about that too. But I left it like this to avoid too many lines. I thought it would be easy enough to  view the list of services and pick and choose from it as it is shown above.